bindings/node/lib/clientEncryption.js

```diff
@@ -549,6 +558,65 @@ module.exports = function (modules) {
       return value;
     }
 
+    /**
+     * A convenience method for creating an encrypted collection.
+     * This method will create data keys for any encryptedFields that do not have a `keyId` defined
+     * and then create a new collection with the full set of encryptedFields.
+     *
+     * @template {TSchema} - Schema for the collection being created
+     * @param {Db} db - A database to create the collection in
+     * @param {string} name - The name of the collection to be created
+     * @param {object} options - Options for createDataKey and for createCollection.
+     * @param {string} options.provider - KMS provider name
+     * @param {AWSEncryptionKeyOptions | AzureEncryptionKeyOptions | GCPEncryptionKeyOptions} [options.masterKey] - masterKey to pass to createDataKey
```

Comment:
this type needs to reflect that we only accept the master key, right?

Reply:
it does, no?

- `options.createCollectionOptions`
- `options.provider`
- `[options.masterKey]`

Reply:
Ah you're correct.